ENIGMAsystem/SHELL/Widget_Systems/Win32/dialogs.cpp

```diff
@@ -17,9 +17,6 @@
 *** with this code. If not, see <http://www.gnu.org/licenses/>
 **/
 
-// Windows Vista or later for IFileDialog
-#define NTDDI_VERSION NTDDI_VISTA
-#define _WIN32_WINNT _WIN32_WINNT_VISTA
```

Comment:
See that's what I was saying about how we have to define it ourselves and should be done in a central location like the SHELL makefile for the entire game. This is kind of an issue of its own though, but as I said then we may also want a literal IDE preference/compiler/game setting for it too.